docker-compose-prod.yml

```diff
@@ -0,0 +1,37 @@
+# docker-compose-prod.yml
+version: "3"
+services:
+  dashboard:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile:0.0.1
+    image: op-admin-dash-prod/opdash
```

Comment:
Ditto

Reply:
solved in https://github.com/e-mission/op-admin-dashboard/pull/7/commits/62307f87190e87e9ee7e390b41681120d79f613f